app/models/employer_group_size_rating_factor_set.rb

```diff
@@ -8,6 +8,6 @@ def self.value_for(carrier_profile_id, year, val)
 
   def lookup(val)
     lookup_key = (val > max_integer_factor_key) ? max_integer_factor_key : val
-    super(lookup_key.to_s)
+    super(lookup_key.to_f.to_s)
```

Comment:
@brianweiner  before the key value is "1" so the rating factor set is returning nil because the factor_key is "1.0".  so changed it to float "1.0".